### DISCUSSION TITLE
How to add more fics to those already downloaded?

### DISCUSSION THREAD

**Discussion by Dalialyng** [nianeyna/ao3downloader#180] [Q&A] [answered]
Let's say I download all the fics I have in "bookmarks" and "mark for later", but then I add more fics to those categories (on the official page). If I wanted to add those new additions to the fics I already downloaded, would repeating the process with which I downloaded the categories in the first place cause duplicates of the fics that were already there? If the answer is yes, what would I have to do to update and add the new fics? would I have to delete and download all the fics again or something?

**Comment by nianeyna** [comment, marked as answer]
Re-downloading will not create duplicate files inside the ao3downloader downloads folder unless you have either changed the ao3downloader naming scheme (using settings.ini) between downloads, OR the identifying details of the work (by default title and author) have changed between downloads. However, if you have moved your files out of the downloads folder to some other location, new copies will be created inside the downloads folder if you do a re-download of the same link.

I hope that answers your question. You can also read over the Notes section of the readme, which has some more detailed explanation of ao3downloader's logic around repeat downloads.

**Reply by Dalialyng** [reply]
Thanks! I'm going to try that.
